build.gradle

```diff
@@ -10,13 +10,6 @@ plugins {
 repositories {
 	mavenCentral()
 	maven { url 'https://repo.spring.io/milestone' }
-
-	maven {
-		url 'https://oss.sonatype.org/content/repositories/snapshots'
-		content {
-			includeGroup 'com.contentgrid.thunx'
-		}
-	}
```

Comment:
any particular reason why you want to remove this ? :shrug:

Reply:
It's not used for any dependencies right now.

We probably shouldn't be using thunx snapshots in our gateway.
For local tests and development with an in-development version, we can always use `includeBuild()`